src/rest-api/spec/training.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
+import {} from 'mocha'
+import {} from 'core-js'
+import { Training } from '../../model/training';
+import { Student } from '../../model/student';
+import { Trainer } from '../../model/trainer';
+import { trainingApi } from '../training'
+
+let trainingList : Training[];
+
+beforeEach(() => {
+  trainingList =
+  [
+    {
+      id: 32,
+      name: 'React/Redux',
+      isActive: true,
+      start: new Date(1,1,2017),
+      end: new Date(31,1,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+    {
+      id: 12,
+      name: 'Responsive web design',
+      isActive: true,
+      start: new Date(1,2,2017),
+      end: new Date(28,2,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+    {
+      id: 33,
+      name: 'AngularJS 2.0',
+      isActive: true,
+      start: new Date(1,3,2017),
+      end: new Date(31,3,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+  ];
+});
+
+describe('TrainingApi', () => {
+  it('is defined', () => {
+      expect(trainingApi).not.to.be.undefined;
+  })
+
+  describe('setMockDataSeed', () => {
+    it('set the right mockdata seed', () => {
+      // Act
+      trainingApi.setMockDataSeed(trainingList);
```

Comment:
The reason behind this, is that we are using a singleton (and the singleton data is feeded in the constructor).

This is just for the mock api, is something temporary, later on we will remove this.

We could add something more complex, like simulating and AJAX call or something like that, but not sure if this applies at this stage.